felix/fv/wireguard_test.go

```diff
@@ -399,6 +402,23 @@ var _ = infrastructure.DatastoreDescribe("_BPF-SAFE_ WireGuard-Supported", []api
 							}, "30s", "330ms").ShouldNot(BeEmpty())
 						}
 					}
+
+					// Make sure napi threading is set properly
+					wireguardThreadingBit := boolToBinaryString(wireguardThreadingEnabled)
+					for _, felix := range topologyContainers.Felixes {
+						if wireguardEnabledV4 {
+							Eventually(func() string {
+								s, _ := felix.ExecCombinedOutput("cat", fmt.Sprintf("/sys/class/net/%s/threaded", ifaceName))
+								return s
+							}, "300s", "5s").Should(ContainSubstring(wireguardThreadingBit))
+						}
+						if wireguardEnabledV6 {
+							Eventually(func() string {
+								s, _ := felix.ExecCombinedOutput("cat", fmt.Sprintf("/sys/class/net/%s/threaded", ifaceNameV6))
+								return s
+							}, "300s", "5s").Should(ContainSubstring(wireguardThreadingBit))
```

Comment:
Do we need to wait up to 300s? What's a lower yet safe period to wait?

Reply:
60s was too short for my dev machine. 120s works.